PurchasesWrapper.java

```diff
@@ -48,7 +48,7 @@ public void onReceived(@NonNull PurchaserInfo purchaserInfo) {
         }
     };
 
-    public static void setup(String apiKey, String appUserId, String gameObject_, boolean observerMode) {
+    public static void setup(String apiKey, String appUserId, String gameObject_, boolean observerMode, string userDefaultsSuiteName) {
         gameObject = gameObject_;
         PlatformInfo platformInfo = new PlatformInfo(PLATFORM_NAME, PLUGIN_VERSION);
         CommonKt.configure(UnityPlayer.currentActivity, apiKey, appUserID, observerMode, platformInfo);
```

Comment:
There's a compilation error here, since it can't find `appUserID`. The parameter name is `appUserId`. Why haven't we noticed this before?

Reply:
Oh actually, this file is in the root of the repository. We should remove it. I see the `appUserId` fix is done in the other `PurchasesWrapper.java`